docs/theory/addition_theorem.rst

```diff
@@ -0,0 +1,63 @@
+#################
+Addition Theorem
+#################
+
+
+**Warning:** you can get by fine without reading this page for almost all use cases, just use the standard :class:`MaternGeometricKernel <geometric_kernels.kernels.MaternGeometricKernel>`, following the example notebook `on hypersheres <https://github.com/GPflow/GeometricKernels/blob/main/notebooks/Hypersphere.ipynb>`_. This is optional material meant to explain the basic theory and based mainly on `Borovitskiy et al. (2020) <https://arxiv.org/abs/2006.10160>`_.
```

Comment:
Here and everywhere, maybe replace "WARNUNG!" with "Disclaimer" or something like this? 

Can we also highlight these warnings with something like `.. warning::` or `.. note::`? I think [this](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) might be relevant. I'd go with the note, since this is not really a warning.